bower.json

```diff
@@ -7,8 +7,8 @@
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
     "Faker": "3.0.1",
-    "lodash": "~4.5.0",
-    "perfect-scrollbar": "~0.6.10",
+    "lodash": "^4.10.0",
+    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```

Comment:
The changes to package.json and bower.json are done by 'ember/npm install'  I did not modify them manually. Should I fix it manually now?